File: ios/BTE/PTCExposureNotificationEventEmitter.m
#import "PTCExposureNotificationEventEmitter.h"

@implementation PTCExposureNotificationEventEmitter

- (instancetype)init
{
  self = [super init];
  if (self) {
    //
  }
  return self;
}

- (NSArray<NSString *> *)supportedEvents {
  return @[@"deleteDiagnosisKeyFile"];
}

@end

File: ios/BTE/bridge/PTCExposureNotificationEventEmitter.swift
import Foundation

@objc(PTCExposureNotificationEventEmitter)
class PTCExposureNotificationEventEmitter: RCTEventEmitter {
  
  public static var emitter: RCTEventEmitter!
  
  override init() {
    super.init()
    PTCExposureNotificationEventEmitter.emitter = self
  }
  
  override func supportedEvents() -> [String]! {
    return ["onExposureNotificationStateUpdated", "onRequestDeleteDiagnosisKeyFile"]
  }
}
